doc/data-files-description.md

```diff
@@ -31,66 +31,66 @@ This document contains information about all data files associated with this pro
+|`pbta-gene-expression-rsem-tpm.stranded.rds` | PBTA data file | [Gene expression abundance estimation](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/content/06.methods.md#gene-expression-abundance-estimation); [Workflow](https://github.com/kids-first/kf-rnaseq-workflow/blob/052e31844711fa5cd0ef5e581618e9d7bba3fec6/workflow/kfdrc_RNAseq_workflow.cwl)  | Gene expression - RSEM TPM for stranded samples (gene-level)
+|`pbta-histologies.tsv` | PBTA data file | [Clinical data harmonization](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/content/06.methods.md#clinical-data-harmonization) | Harmonized clinical metadata file (see data dictionary [here](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/content/06.methods.md#clinical-data-harmonization))
+|`pbta-isoform-counts-rsem-expected_count.polya.rds` | PBTA data file | [Gene expression abundance estimation](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/content/06.methods.md#gene-expression-abundance-estimation); [Workflow](https://github.com/kids-first/kf-rnaseq-workflow/blob/052e31844711fa5cd0ef5e581618e9d7bba3fec6/workflow/kfdrc_RNAseq_workflow.cwl) | Gene expression - RSEM expected counts for poly-A samples (transcript-level)
+|`pbta-isoform-counts-rsem-expected_count.stranded.rds` | PBTA data file | [Gene expression abundance estimation](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/content/06.methods.md#gene-expression-abundance-estimation); [Workflow](https://github.com/kids-first/kf-rnaseq-workflow/blob/052e31844711fa5cd0ef5e581618e9d7bba3fec6/workflow/kfdrc_RNAseq_workflow.cwl) |Gene expression - RSEM expected counts for stranded samples (transcript-level)
+|`pbta-isoform-expression-rsem-tpm.polya.rds` | PBTA data file | [Gene expression abundance estimation](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/content/06.methods.md#gene-expression-abundance-estimation); [Workflow](https://github.com/kids-first/kf-rnaseq-workflow/blob/052e31844711fa5cd0ef5e581618e9d7bba3fec6/workflow/kfdrc_RNAseq_workflow.cwl) | Gene expression - RSEM TPM for poly-A samples (transcript-level)
+|`pbta-isoform-expression-rsem-tpm.stranded.rds` | PBTA data file | [Gene expression abundance estimation](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/content/06.methods.md#gene-expression-abundance-estimation); [Workflow](https://github.com/kids-first/kf-rnaseq-workflow/blob/052e31844711fa5cd0ef5e581618e9d7bba3fec6/workflow/kfdrc_RNAseq_workflow.cwll) | Gene expression - RSEM TPM for stranded samples (transcript-level)
```

Comment:
The change in section header need to be addressed here as well.

Reply:
`v21` indeed!